modules/nw-egress-router-about.adoc

```diff
@@ -37,7 +37,12 @@ The egress router image is not compatible with Amazon AWS, Azure Cloud, or any o
 [id="nw-egress-router-about-modes_{context}"]
 == Egress router modes
 
-In _redirect mode_, an egress router pod configures `iptables` rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be modified to connect to the egress router rather than connecting directly to the destination IP.
+In _redirect mode_, an egress router pod configures `iptables` rules to redirect traffic from its own IP address to one or more destination IP addresses. Client pods that need to use the reserved source IP address must be configured to access the service for the egress router rather than connecting directly to the destination IP. The following example demonstrates how to access the destination service and port from the application pod:
+
+[source, terminal]
+----
+$ curl <router service IP> <port>
```

Comment:
The [supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values) states the following for user-replaced values:
> Separated by underscores (_) for multi-word values
```
s/<router service IP>/<router_service_IP>
```

Reply:
Darn, I missed these. Thanks. Updated.